templates/index.html

```diff
@@ -1,8 +1,57 @@
-{% extends 'base.html' %}
-
-{% block head %}
+{% extends 'base.html' %}{% block head %}
 <title>Home</title>
+<!--Fonts-->
+<link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@800&display=swap" rel="stylesheet">
 {% endblock %}
 
 {% block body %}
+<div class="container">
+    <div class="row">
+        <div class="col-md-6">
+            <p class="roundb font-weight-normal position-absolute"> Grab yours now! </p>
+            <h1 class="font1 mt-4">Enjoy Great Deals of Beauty Essentials!</h1>
+            <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Voluptatum voluptatem molestias laborum doloribus recusandae ex nam inventore nobis sapiente quasi? Minima quaerat ullam sapiente laudantium fuga quas vero distinctio aliquam?</p>
+            <h6 class="btn btn-main">Shop Now</h6>
+        </div>
+        <div class="col-md-3">
+            eyyyy
+        </div>
+        <div class="col-md-3">
+            another one
+        </div>
+    </div>
+</div>
+
+<div class="container shadow p-3 mb-5 mt-5 bg-white rounded-lg">
```

Comment:
pwede mo man makakas ang 'rounded-lg'. Replace with this code instead para mas nami pa iya effect.
`style="border-radius: 40px;"`